src/pages/using-spark/components/button.mdx

```diff
@@ -115,14 +76,17 @@ Represents the second-highest priority action, following the Primary Button. The
   titleAttr="Secondary Button Example"
 />
 
+- There can be more than one on a screen.
+- Use for alternative actions that do not navigate the user to the next step in a process.
+
 ### Tertiary Button
-Represents less prominent actions. Sometimes in an action set with [Primary](#primary-button) and [Secondary](#secondary-button) Buttons, but they can also be independent actions.
 
-#### Example
-- In a multi-step form, a Tertiary Button works with Primary and Secondary Button as a set of related actions.
-  - Primary Button is "Submit"
-  - Secondary Button is "Back"
-  - **Tertiary Button is "Cancel"**
+It represents the least important action on a screen. They have a similar visual style to a text [Link](/using-spark/components/link) and
```

Comment:
Remove "it represents"

"The least important action on the the **page**." 
not screen